### PR TITLE
Swap all repeats to multiple='True' where appropriate

### PR DESCRIPTION
**Pull request by erasche**
In support of https://github.com/galaxyproject/tools-iuc/issues/239

- [x] Depends on galaxyproject/galaxy#749

### CODE REVIEW

File: tool_collections/bamtools/bamtools/bamtools.xml [hunk 1]
```diff
@@ -1,19 +1,19 @@
-<?xml version="1.0"?>
+<?xml version="1.0.1"?>
```
Comment (bgruening) [inline tool_collections/bamtools/bamtools/bamtools.xml]: What is that? ;)

File: tool_collections/bamtools/bamtools/bamtools.xml [hunk 2]
```diff
@@ -27,83 +27,81 @@
...
+
+
   </command>
   <inputs>
-    
-    <repeat name="input_bams" title="BAM dataset(s) to filter" min="1">
-          <param name="input_bam" type="data" format="bam" label="BAM dataset" />
-    </repeat>
+
+    <param name="input_bams" type="data" format="bam" label="BAM dataset(s) to filter" min="1"/>
```
Comment (bgruening) [inline tool_collections/bamtools/bamtools/bamtools.xml]: multiple="True"?

File: tool_collections/bamtools/bamtools/bamtools.xml (hunk 1 above)
Comment (erasche) [inline tool_collections/bamtools/bamtools/bamtools.xml]: patch version? I guess I can change to 1.1

File: tool_collections/bamtools/bamtools/bamtools.xml [hunk 3]
```diff
@@ -1,19 +1,19 @@
-<?xml version="1.0"?>
+<?xml version="1.1"?>
```
Comment (bgruening) [inline tool_collections/bamtools/bamtools/bamtools.xml]: This should not be changed, it's the XML version not the tool version :)

File: tool_collections/vcftools/vcftools_merge/vcftools_merge.xml [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
     </requirements>
-    
+
     <command>
         ## Preprocessing for each dataset.
         #set dataset_names = []
-        #for i, $input in enumerate( $inputs ):
-            ## Sort file.         
-            vcf-sort ${input.file} > ${i}.vcf.sorted ;
+        #for i, $input in enumerate( $input_files ):
```
Comment (bgruening) [inline tool_collections/vcftools/vcftools_merge/vcftools_merge.xml]: Why does the `i` does not have a $.

File: tool_collections/samtools/samtools_mpileup/samtools_mpileup.xml [hunk 5]
```diff
@@ -18,8 +18,8 @@
     #else:
         -f "${reference_source.ref_file}"
     #end if
-    #for $i, $input_bam in enumerate( $reference_source.input_bams ):
-        "${input_bam.input_bam}"
+    #for $i, $bam  in enumerate( $reference_source.input_bam ):
```
Comment (bgruening) [inline tool_collections/samtools/samtools_mpileup/samtools_mpileup.xml]: enumerate is not needed.

File: tool_collections/vcftools/vcftools_merge/vcftools_merge.xml (hunk 4 above)
Comment (erasche) [inline tool_collections/vcftools/vcftools_merge/vcftools_merge.xml]: Input doesn't need it either. That's a bug

File: tool_collections/samtools/samtools_mpileup/samtools_mpileup.xml (hunk 5 above)
Comment (erasche) [inline tool_collections/samtools/samtools_mpileup/samtools_mpileup.xml]: fixing...

**Comment by jmchilton** [comment]
Looks fine to me, these are all pretty recent tools. I want to ping @nekrut though, he implemented a lot of these and may have had reasons for using repeats.

**Comment by bgruening** [comment]
ping @nekrut 
We can merge this now. You have the last word :)
Any concerns in backward compatibility?

**Pull request merged**